modules/new_profile/jsx/newIndex.js

```diff
@@ -0,0 +1,20 @@
+import NewProfileForm from './NewProfileForm';
+ class NewIndex extends React.Component {
+        render() {
+                return (
+                  <div>
+              <NewProfileForm />
+                  </div>
+                );
```

Comment:
Lines `3-8` can be condensed to 1 line. It'll make reading the code a bit easier.

Reply:
done